modules/fluviewer.nf

```diff
@@ -56,6 +85,7 @@ process fluviewer {
 	--min-identity ${params.min_ident} \
 	--max-memory 40 \
 	--disable-garbage-collection \
+	--skip-depth-normalization \
```

Comment:
This pertains to `fluviewer`, not `fluviewer-nf`, but they are linked here. Do we want the default `fluviewer` behaviour to be to normalize? This is really minor, but it flows a bit better in my head to have the simpler behaviour (no normalize) be the default. What are your thoughts?

Reply:
We want the default behaviour of `BCCDC-PHL/fluviewer` to match what the original version does. This way we maintain consistency, while still gaining the ability to move bbnorm to a separate, dedicated Nextflow process

Reply:
Hi @jpalmer37, I was thinking that since we're still aiming to maintain consistency with [KevinKuchinski/FluViewer](https://github.com/KevinKuchinski/FluViewer) v0.1.11, I thought it would be best to keep our default behavior the same as that tool.